cloudbuild/python-scripts/utils/utils.py

```diff
@@ -0,0 +1,184 @@
+"""Python class containing basic utilities required for the creation of BQ Table.
+
+Can create specified number of rows, transfer them to BQ, delete the local file.
+"""
+
+import datetime
+import os
+import re
+import avro
+import avro.datafile
+import avro.io
+from google.cloud import bigquery
+
+
+class TableCreationUtils:
+    """Class containing all the basic utilities for creation of BQ table.
+
+    Attributes:
+      table_type: Table type to be created: can be 'large_table',
```

Comment:
`table_type: Table type to be created. Can be ...` Replace colon with full stop.

Reply:
Also, lets use enums instead of strings for `table_type`

Reply:
Ok, I see that TableType class is defined in next PR. You should get that reviewed first, or combine both changes into 1 PR. Current order is incorrect since #73 lacks context without #74.

Reply:
Added table_type in the PR as well.